docs/auth.md

```diff
@@ -0,0 +1,149 @@
+# Introduction
+
+kcp implements the RBAC based authorization scheme as Kubernetes.
+Other authorization schemes (i.e. ABAC) are currently not supported.
+Generally, the same (cluster) role and (cluster) role binding priniciples apply the same way as in Kubernetes.
+
+However, since kcp is based around workspaces and workspaces are forming a tenancy boundary,
+RBAC rules are always declared within the context of a concrete workspace.
+These rules apply within the workspace they were declared and do not apply in other workspaces.
+The type of the workspace doesn't matter, it can either be the enduser, organizational, or a root workspace.
+The exception are system namespaces. These can only be accessed by shard-local admins.
+
+# Authorizers
+
+## Workspace content authorizer
+
+As described above, RBAC rules apply for resources within a workspaces only.
+They cannot express the basic authorization primitive whether a subject is generally
+authorized to access a workspace.
+
+The lack of any bindings is not sufficient to express general access to a workspace
+as every authenticated user in Kubernetes is at least bound to `system:authenticated`
+with bindings against at least some cluster roles, like `system:basic-user`, `system:discovery`,
+and `system:public-info-viewer`.
+
+Expressing basic access to a workspace via bindings against the `ClusterWorkspace` resource type
+is not sufficient as well because the permission to mutate the `ClusterWorkspace` resource itself
+(i.e. changing its metadata) is an orthogonal use case to accessing its contained resources.
+
+Hence, an additional authorization layer is necessary. To express authorization patterns
+kcp recognizes role bindings defined against the `workspaces/content` resource under the `tenancy.kcp.dev` api group.
+This subresource is not backed by any CRD but is known to the workspace content authorizer only.
+
+Since `ClusterWorkspaces` are defined in organization workspaces only
+`workspaces/content` role bindings are expected to be contained in organizational workspaces as well.
+Concrete permissions can be declared using verbs.
+
+The workspace content authorizer ensures that permitted verbs are mapped to groups for a given subject.
+The assigned group is by default bound against standard Kubernetes cluster roles
+such that a simple binding against the `workspace/content` resource
```

Comment:
another instance of workspaces that should change to clusterworkspaces

Reply:
fixed